scripts/gz3d/gz3d.py

```diff
@@ -76,17 +76,13 @@ def _info(self):
         # Starting with all features common to image datasets
         features = {
             # Metadata
-            'total_classifications': Value("int16"),
-            'healpix': Value("int16"),
-            'ra': Value('float32'),
-            'dec': Value('float32'),
+            'total_classifications': Value("uint8"),
             # Image data
-            'image': Array2D(shape=(self._image_size, self._image_size, 3), dtype='int16'),
-            'scale': Value('float32'),
+            'image': Array2D(shape=(self._image_size, self._image_size), dtype='uint8'),
             'segmentation': Sequence(feature={
                 'class': Value('string'),
                 'vote_fraction': Value('float32'),
-                'array': Array2D(shape=(self._image_size, self._image_size), dtype='float32')
```

Comment:
I think we should include all of these except healpix (I like the new dtypes more).